Formula/openjdk.rb

```diff
@@ -0,0 +1,68 @@
+class Openjdk < Formula
+  desc "Development kit for the Java programming language"
+  homepage "https://openjdk.java.net/"
+  url "https://hg.openjdk.java.net/jdk-updates/jdk13u/archive/jdk-13.0.1+9.tar.bz2"
+  version "13.0.1+9"
+  sha256 "97328e767bc5f47b097ec0e9d88a6a650e60c448dbaba2e835284a2bf5594eb5"
+
+  keg_only :provided_by_macos
+
+  depends_on "autoconf" => :build
+  depends_on "openjdk@12" => :build
+  depends_on "freetype"
+
+  def install
+    ENV["JAVA_HOME"] = Formula["openjdk@12"].libexec/"openjdk-12.jdk/Contents/Home"
+    java_options = ENV.delete("_JAVA_OPTIONS")
+
+    inreplace "make/autoconf/lib-freetype.m4",
```

Comment:
Has this been proposed upstream? Why is the reason they won't allow a non-bundled freetype?

Reply:
Seems like the reason is “building freetype separately is a pain”.

https://bugs.openjdk.java.net/browse/JDK-8193017

Reply:
Removed this for now.